lib/manager/pip_requirements/index.ts

```diff
@@ -1,7 +1,7 @@
 import { LANGUAGE_PYTHON } from '../../constants/languages';
 
 export { updateArtifacts } from './artifacts';
-export { extractPackageFile } from './extract';
+export { extractAllPackageFiles } from './extract';
```

Comment:
@rarkins  You need to export both functions to get it working with auto-replace

Reply:
Thanks. I decided to revert it for now as the quickest resolution